src/components/Thumbnail.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Video from "./Video";
+
+const videoThumb = {
+  flex: "0 1 30vw",
+  width: "30vw",
+  margin: "0 auto",
+  display: "flex",
+  flexDirection: "column",
+  alignItems: "center",
+};
+
+const Thumbnail = (props) => {
+  function getCanvasThumbnail(link) {
+    let thumbnail; //= canvas X html5 X link
+    return [thumbnail];
+  }
+  const [thumbnail] = getCanvasThumbnail(props.video.location);
+
+  return (
+    <div >
```

Comment:
We also probably want to enforce using <> </> (react fragments: https://reactjs.org/docs/fragments.html)

I see a mix of us using them and not using them, with no real rhyme or reason